src/ProofOfConcepts/ConnectorMessageDrawer/DrawerSplit/ConnectorDrawerMessageStatistics.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { FunctionComponent } from "react";
+
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  CardTitle,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Divider,
+  Flex,
+  FlexItem,
+  Grid,
+  GridItem,
+  Split,
+  SplitItem,
+  Text,
+  TextContent,
+  TextListItem,
+  TextListItemVariants,
+  TextVariants,
+  Title,
+} from "@patternfly/react-core";
+import {
+  global_success_color_100,
+  global_danger_color_100,
+} from "@patternfly/react-tokens";
+
+import { ExclamationIcon } from "@patternfly/react-icons";
+import CheckIcon from "@patternfly/react-icons/dist/esm/icons/check-icon";
+
+import "./ConnectorDrawer.css";
+
+export type ConnectorDrawerMessageStatisticsProps = {
+  sent: string;
+  notSent: string;
+};
+
+export const ConnectorDrawerMessageStatistics: FunctionComponent<
+  ConnectorDrawerMessageStatisticsProps
+> = ({ sent, notSent }) => {
+  return (
+    <Flex direction={{ default: "column" }}>
+      <FlexItem>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h4">Processed messages</Title>
+          </CardTitle>
+          <CardBody>
+            <Split>
+              <SplitItem isFilled>
+                {/** displaysize in description list will be better use then utilites classes, but throwing errors for some reason */}
+                <Flex
+                  className="pf-u-font-size-2xl"
+                  direction={{ default: "row" }}
```

Comment:
I don't think the Flex layout includes a prop to prevent wrapping, so I think you also need to include the utility class `pf-u-flex-nowrap` on the flex layouts here. This keeps the first line of text in the same line as the icon, and forces only the text to wrap, rather than moving the whole text block underneath the icon.